### PR TITLE
add Vertamedia video adpater

### PR DESCRIPTION
**Pull request by Gorokhov**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [X ] New bidder adapter
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other

## Description of change
Adding support of vertamedia prebid

- test parameters for validating bids
```
{
  bidder: 'vertamedia',
  params: {
           aid: 22489,
           placementId: '123456'
  }
}
```
- contact email of the adapter’s maintainer
y.gorokhov@verta.media
m.sorochuk@veta.media
- [X ] official adapter submission

### CODE REVIEW

File: src/adapters/vertamedia.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    try {
+      parsed = JSON.parse(response);
+    } catch (error) {
+      utils.logError(error);
+    }
+
+    if (!parsed || parsed.error || !parsed.bids || !parsed.bids.length) {
+      utils.logError(`in response for ${baseAdapter.getBidderCode()} adapter`);
+      bidmanager.addBidResponse(bidRequest.placementCode, createBid(STATUS.NO_BID));
+
```
Comment (jaiminpanchal27) [inline src/adapters/vertamedia.js]: Remove extra line

File: src/adapters/vertamedia.js [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+      return bid;
+    }
+
+    bid.mediaType = 'video';
+    bid.cpm = tag.cpm;
+    bid.creative_id = tag.cmpId;
+    bid.width = bidRequest.width;
+    bid.height = bidRequest.height;
+    bid.vastUrl = tag.url;
+
```
Comment (jaiminpanchal27) [inline src/adapters/vertamedia.js]: Add one more property descriptionUrl to bid object. We made some change in adserver.js recently. Do not remove vastUrl yet.
Check #967

File: src/adapters/vertamedia.js (hunk 1 above)
Comment (Millerrok) [inline src/adapters/vertamedia.js]: @jaiminpanchal27 done

File: src/adapters/vertamedia.js (hunk 2 above)
Comment (Millerrok) [inline src/adapters/vertamedia.js]: @jaiminpanchal27 done

**Comment by Gorokhov** [comment]
Hey @jaiminpanchal27 , can i get an approx estimate when we become visible here:
http://prebid.org/download.html
I need this in order to define a date for a press release for our marketing team.

**Comment by jaiminpanchal27** [comment]
@Gorokhov We are planning to release 0.19.0 early next week. 
It will be on [Download](http://prebid.org/download.html) page, approx by March 15th.

**Comment by jaiminpanchal27** [comment]
This is merged into master. Please submit a PR to the [docs repo](https://github.com/prebid/prebid.github.io) to add a file for your adapter to the [bidders directory](https://github.com/prebid/prebid.github.io/tree/master/dev-docs/bidders) so your adapter's params will appear on the [bidders page](http://prebid.org/dev-docs/bidders.html). Thank you for contributing

**Pull request merged**